ci-integration/virtualization/Dockerfile

```diff
@@ -3,21 +3,20 @@ ARG image_repo=volttron-docker
 ARG image_tag=deps-master-amd64
 
 FROM ${image_user}/${image_repo}:${image_tag} as volttron_deps
-
```

Comment:
changing the image_tag to deps-feature_python3-amd64

makes the user root stuff below irrelevent.